doc/analysis-plan.md

```diff
@@ -0,0 +1,148 @@
+    ##  2               2     ATSH080               2 Rhizoprionodon terraenovae
+    ##  3               3     ATSH089               3 Rhizoprionodon terraenovae
+    ##  4               4     ATSH143               4 Rhizoprionodon terraenovae
+    ##  5               5     ATSH161               5 Rhizoprionodon terraenovae
+    ##  6               6     ATSH166               6 Rhizoprionodon terraenovae
+    ##  7               7     ATSH172               7 Rhizoprionodon terraenovae
+    ##  8               8     ATSH192               8 Rhizoprionodon terraenovae
+    ##  9               9     ATSH205               9 Rhizoprionodon terraenovae
+    ## 10              10     ATSH205               9 Rhizoprionodon terraenovae
+    ## # ... with 34,921 more rows, and 56 more variables: `Predator common
+    ## #   name` <chr>, Predator_taxon <chr>, `Predator lifestage` <chr>,
+    ## #   feeding_type <chr>, `Predator length` <dbl>, `Predator length
+    ## #   unit` <chr>, `Predator dimension measured` <chr>,
+    ## #   predator_standars_length <dbl>, `Predator fork length` <dbl>,
+    ## #   `Predator total length` <dbl>, `Predator TL/FL/SL conversion
+    ## #   reference` <chr>, `Standardised predator length` <dbl>, `Predator
+    ## #   measurement type` <chr>, `Predator length-mass conversion
+    ## #   method` <chr>, `Predator length-mass conversion reference` <chr>,
+    ## #   `Predator quality of length-mass conversion` <dbl>,
+    ## #   predator_mass <dbl>, `Predator mass unit` <chr>, `Predator mass
+    ## #   check` <dbl>, `Predator mass check diff` <dbl>, `Predator ratio
+    ## #   mass/mass` <dbl>, `SI predator mass` <dbl>, `Diet coverage` <chr>,
+    ## #   Prey <chr>, `Prey common name` <chr>, `Prey taxon` <chr>, `Prey
+    ## #   length` <dbl>, `Prey length unit` <chr>, `Prey conversion to length
+    ## #   method` <chr>, `Prey quality of conversion to length` <dbl>, `Prey
+    ## #   conversion to length reference` <chr>, `SI prey length` <dbl>, `Prey
+    ## #   dimension measured` <chr>, `Prey width` <chr>, `Prey width
+    ## #   unit` <chr>, `Prey measurement type` <chr>, prey_mass <dbl>, `Prey
+    ## #   mass unit` <chr>, `Prey mass check` <dbl>, `Prey mass check
+    ## #   diff` <dbl>, `Prey ratio mass/mass` <dbl>, `SI prey mass` <dbl>, `Prey
+    ## #   conversion to mass method` <chr>, `Prey conversion to mass
+    ## #   reference` <chr>, `Prey quality of conversion to mass` <dbl>,
+    ## #   geographic <chr>, Latitude <chr>, Longitude <chr>, Depth <dbl>,
+    ## #   temp <dbl>, `SD annual temp` <dbl>, `Mean PP` <dbl>, `SD PP` <dbl>,
+    ## #   Reference <chr>, habitat <chr>, `Notes / assumptions` <chr>
+
+    g <- ggplot(marine2, color = predator_taxon)
+    g+
+      geom_boxplot(aes(x = feeding_type, y = predator_mass))+
+      ylim(0,5000)
+
+    ## Warning: Removed 11279 rows containing non-finite values (stat_boxplot).
+
+# Possible results figures
+
```

Comment:
Also, while the figures are great, please add some text briefly describing some *other* figures you could include.